README.md

```diff
@@ -1,148 +1,43 @@
-# purescript-uri
+# uri
 
-[![Latest release](http://img.shields.io/github/release/purescript-contrib/purescript-uri.svg)](https://github.com/purescript-contrib/purescript-uri/releases)
-[![Build status](https://travis-ci.org/purescript-contrib/purescript-uri.svg?branch=master)](https://travis-ci.org/purescript-contrib/purescript-uri)
+[![CI](https://github.com/purescript-contrib/purescript-uri/workflows/CI/badge.svg?branch=main)](https://github.com/purescript-contrib/purescript-uri/actions?query=workflow%3ACI+branch%3Amain)
+[![Release](http://img.shields.io/github/release/purescript-contrib/purescript-uri.svg)](https://github.com/purescript-contrib/purescript-uri/releases)
+[![Pursuit](http://pursuit.purescript.org/packages/purescript-uri/badge)](http://pursuit.purescript.org/packages/purescript-uri)
+[![Maintainer: garyb](https://img.shields.io/badge/maintainer-garyb-teal.svg)](http://github.com/garyb)
 
-A type-safe parser, printer, and ADT for URLs and URIs based on [RFC 3986](http://tools.ietf.org/html/rfc3986).
+The library summary hasn't been written yet (contributions are welcome!). The library summary describes the library's purpose in one to three sentences.
```

Comment:
We should preserve this library summary instead of falling back to the placeholder text. For libraries which have no summary, part of the contrib update needs to be writing a short summary. For libraries which do have one (which includes this library) we should at least preserve the existing summary and (if possible) improve it.

In this case we can preserve the existing summary:

> A type-safe parser, printer, and ADT for URLs and URIs based on [RFC 3986](http://tools.ietf.org/html/rfc3986).

Reply:
Whoops! I preserved that originally before I played around with the `contrib-updater` tool.